public/controllers/AppController.js

```diff
@@ -12,4 +12,23 @@ contactListApp.controller("ContactTableController", function($scope, $http)
             return vm;
         }
     );
+    
+    $scope.addContact = function()
+    {
+        $http.post("/contacts", $scope.contact).then
+        (
+            (
+                function(response)
+                {
+                    vm.contacts.push(response.data);
+                    
+                    $scope.contact.name = "";
```

Comment:
This is the code for clearing the input fields. I want to move away from using `$scope` since I am using the new "controller as x" syntax, but I plan on doing that in the next ticket anyways.